database/index.js

```diff
@@ -3,36 +3,78 @@ mongoose.connect('mongodb://localhost:27017/moodu');
 
 
 /* SCHEMAS */
-let reviewsSchema = mongoose.Schema({
-    id: 'number',
-    filmname: 'string',
-    reviews: [
-      {
-        id: 'number',
-        review: 'string',
-        author: 'string',
-        rating: 'number',
-        source: 'string'
-      }
-    ]
-  });
-
-
-let productsSchema = mongoose.Schema({
-  index: 'number',
+const reviewsSchema = new mongoose.Schema({
+  id: 'number',
+  filmname: 'string',
+  length: 'number',
+  released: 'string',
+  rating: 'number',
+  studio: 'string',
+  language: 'string',
+  uhd: 'string',
+  hdx: 'string',
+  sd: 'string',
+  cc: 'string',
+  reviews: [
+    {
+      id: 'number',
+      review: 'string',
+      author: 'string',
+      rating: 'number',
+      source: 'string'
+    }
+  ]
+});
+
+const  productsSchema = new mongoose.Schema({
+  id: 'number',
   prodname: 'string',
-  price: 'string',
-  added: 'string',
+  price: 'number',
   picture: 'string',
   about: 'string'
 }
 );
 
+
 /* MODELS */
-let Review = mongoose.model('Review', reviewsSchema);
-let Product = mongoose.model('Product', productsSchema);
+const  Review = mongoose.model('Review', reviewsSchema);
+const  Product = mongoose.model('Product', productsSchema);
+
+
+
+async function getReviewsforFilm(filmname)  {

```

Comment:
Using `.exec()` (instead of `async`/`await`) with Mongoose will return a promise - otherwise, these Mongoose methods return _thenables_ which should also work here.